backend/resources/reviews.py

```diff
@@ -76,3 +76,21 @@ def delete(self, user_id, isbn):
 
         return {"message": f"Review with ['user_id': {user_id}, 'isbn': {isbn}] deleted"}, 200
 
+    @auth.login_required()
+    def put(self, user_id, isbn):
+        data = parse_review(False)
+        with lock:
+            user = UsersModel.find_by_id(user_id)
+            check_user_and_book(user, isbn)
```

Comment:
Un admin no hauria de ser capaç de modificar les reviews d'altres persones, crec. Ho comentem a la daily.

Reply:
aixo es lo que he comentat, tindria sentit que si perque tal vegada es neccesari la modificacio per un insult.

Reply:
Solved in https://github.com/UB-ES-2020-A/Grup-ES/pull/205/commits/730461012aeceb53e6171aaff2d479e503928fae.